media-libs/jbig2dec/jbig2dec-0.17.recipe

```diff
@@ -59,6 +59,8 @@ INSTALL()
 {
 	make install
 
+	fixPkgconfig
```

Comment:
Shouldn't this come after prepareInstallDevelLib? (PS it's only one lib so you can remove the extra "s" there :)